wallabagger/js/common.js

```diff
@@ -16,8 +16,11 @@ const Common = (() => {
         });
     };
 
+    const getLocale = () => browser.i18n.getUILanguage();
+
     return {
         'translate': translate,
-        'translateAll': translateAll
-    }
+        'translateAll': translateAll,
```

Comment:
Couldn't you just use shorthand properties here ?

Reply:
yeah. that would be more clear, but let's change it id Dev branch after this PR